src/Umbraco.Web/UmbracoModule.cs

```diff
@@ -605,8 +604,7 @@ public void Init(HttpApplication app)
                 if (UmbracoContext.Current != null)
                 {
                     Logger.Verbose<UmbracoModule>(
-                        "End Request [{RequestId}]: {RequestUrl} ({RequestTotalMilliseconds}ms)",
-                        requestId,
+                        "Request: {RequestUrl} took {Duration}ms",
```

Comment:
See comment above. In fact... if you get rid of RequestId here and let the RequestId enricher deal with it, how will it show in the text log? In other words where would the RequestId show, if it's not part of the logged string but added by the enricher? I understand it would show in the JSON blob but what about text?

Reply:
Good point - will add it back in. I hope people will stop using the old and hard to read .txt file and swap to use the built in logviewer work I am doing, to help them search and query logs instead.